.github/workflows/scripts/e2e-utils.sh

```diff
@@ -80,6 +80,14 @@ _e2e_verify_query() {
     e2e_assert_eq "${name}" "${expected}" "${query} should be ${expected}"
 }
 
+# _e2e_verify_presence verifies that the result of the given jq query is present.
+_e2e_verify_presence() {
+    local attestation="$1"
+    local query="$2"
+    name=$(echo -n "${attestation}" | jq -c -r "${query}")
+    assert_true "${name}" "${query} should be ${expected}"
```

Comment:
IIUC `assert_true` is going to check for the JS boolean value `true` (or string value `"true"`). You probably want to check that the value is empty.

Also `jq` prints "null" when the value isn't present so you can add another check for the return value's type != null.

```suggestion
    name=$(echo -n "${attestation}" | jq -c -r "${query}  | select(type != \"null\")")
    assert_not_empty "${name}" "${query} should be ${expected}"
```